### PR TITLE
Feat/challenge 03 loops conditionals

### PR DESCRIPTION
**Pull request by aalvar0**

### CODE REVIEW

File: challenge_02_variables-operators/index.html [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
-<script src="/js/exercise-04.js"></script>
-
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JavaScript Basics</title>
+  </head>
+  <!-- <script src="/js/exercise-00.js"></script>
```
Comment (tictools) [inline challenge_02_variables-operators/index.html]: ⚠️ FIX: remove commented code

File: challenge_03_loops-conditionals/03_JavaScript_foundations/README.md
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/README.md:1]: `README.md` should live at project root level.

⚠️ FIX: move this file at root level.

File: challenge_03_loops-conditionals/03_JavaScript_foundations/index.js
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/index.js:1]: ⚠️ FIX: remove file

File: challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/conditionals/exercise_00.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    6. If it is neither "cat" nor "dog", print out "Must be an alien"
+    7. At the end of the program print out: "--- Animal check complete ---"
+    8. Change animal to different values to make sure it works in all cases
+*/
+
+console.log("--- Animal Checker Program ---");
+
+let animal = "koala";
+
+if (animal == "cat") {
```
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/conditionals/exercise_00.js]: ⚠️ FIX: Change animal to different values to make sure it works in all cases

File: challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/conditionals/exercise_01.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+    Change the variable to test it works for different cases.
+    Print result.
+
+    HINT: modulus operator ( % ) is your friend
+*/
+
+let number=5;
+
+if (number % 2 === 0) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/conditionals/exercise_01.js]: ⚠️ FIX: Change the variable to test it works for different cases. Print result.

File: challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/conditionals/exercise_02.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  console.log("You are in elementary school, kid.");
+} else if (age >= 13 && age < 18) {
+  console.log("Nice, you're a teenager!");
+} else if (age >= 18) {
+  console.log("Most places consider you an adult.");
+} else {
+  console.log("What? How did this happen!?");
+}
+
+/*The issue of this code is the structure, the program executes the corresponding block and then exits the entire if-else structure.*/
```
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/conditionals/exercise_02.js]: ⚠️ FIX: needs work

File: challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/loops/exercise_00.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+/*
+    1. Create a for loop that will print out all the even numbers between 10 and 40.
+    
+    2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+
+for (let i = 10; i <= 40; i += 2) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/loops/exercise_00.js]: These loops should iterate over all numbers in between 10 and 40, printing only the odd/even ones, depending on its own logic.

⚠️ FIX: iterate over all numbers and print only the odd/even ones

You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

File: challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/loops/exercise_02.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    Half way there!
+    Checkpoint! 60
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+
+for (let i = 1; i <= 100; i++) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/loops/exercise_02.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

File: challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/while/exercise_01.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
+/*
+    1. Create a while loop that will print out all the even numbers between 10 and 40.
+
+    2. Create a while loop that will print out all the odd numbers between 10 and 40.
+*/
+
+console.log("Even numbers between 10 and 40:");
+let even = 10;
+while (even <= 40) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/while/exercise_01.js]: These loops should iterate over all numbers in between 10 and 40, printing only the odd/even ones, depending on its own logic.

⚠️ FIX: iterate over all numbers and print only the odd/even ones

File: challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/while/exercise_02.js [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+
+let counter = 1;
+
+while (counter <= 100) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/while/exercise_02.js]: This is the output of your script
```
Checkpoint! 10
Checkpoint! 20
Checkpoint! 30
Checkpoint! 40
Checkpoint! 50
Half way there!
Checkpoint! 60
Checkpoint! 70
Checkpoint! 80
Checkpoint! 90
Checkpoint! 100
You made it!
All, done!
```

have you tested it before pushing? 🤔

⚠️ FIX: needs work.

**Review by adrianagrd — APPROVED** [review]

File: package.json [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+    "lint": "npx prettier . --check",
+    "lint:fix": "npx prettier . --write"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "prettier": "3.0.3"
+  }
+}
```
Comment (tictools) [inline package.json]: Format code

**Review by tictools — CHANGES_REQUESTED** [review]

File: .github/workflow/pipeline.yaml
Comment (adrianagrd) [inline .github/workflow/pipeline.yaml:1]: The folder must be defined as `.github/workflows/pipeline.yaml` not `.github/workflow/pipeline.yaml`

**Review by adrianagrd — CHANGES_REQUESTED** [review]

File: challenge_03_loops-conditionals/03_JavaScript_foundations/README.md
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/README.md:1]: This file should not live here. It should be placed at project root level.

FIX: remove this file from folder challenge_03 to root

File: README.md
Comment (tictools) [inline README.md:1]: This file is outdated. You should update with README attached in our chat some days ago.

FIX: update file

**Review by tictools — CHANGES_REQUESTED** [review]

**Review by tictools — APPROVED** [review]

**Review by adrianagrd — APPROVED** [review]

**Pull request merged**